core/src/test/java/io/questdb/griffin/engine/functions/regex/LikeFunctionFactoryTest.java

```diff
@@ -278,4 +278,53 @@ public void testNotLikeStringMatch() throws Exception {
             }
         });
     }
```

Comment:
Would be good to test with \ appearing at end of like pattern .

Reply:
Thanks for your suggestion. New test case added and codes got reformatted. Please review. :)